docs/Shell-Completion.md

```diff
@@ -2,7 +2,7 @@
 
 Homebrew comes with completion definitions for the `brew` command. Some packages also provide completion definitions for their own programs.
 
-`zsh`, `bash` and `fish` are currently supported. (Homebrew provides `brew` completions for `zsh` and `bash`; `fish` provides its own `brew` completions.)
+`zsh`, `bash` and `fish` are currently supported. (Homebrew provides `brew` completions for `zsh`, `bash`, and `fish`.)
```

Comment:
I think we can remove the part in parentheses altogether.

Reply:
Good idea. Done!